src/odemis/gui/model/__init__.py

```diff
@@ -1151,20 +1172,23 @@ def _posToCCD(self, posphy):
         res = self.main.ccd.shape[0:2]
 
         # Convert into px referential (Y is inverted)
-        posc_px = (posphy[0] / pxs[0], -posphy[1] / pxs[1])
-        # Convert into the referential with the top-left corner as origin
-        posccd = (posc_px[0] + (res[0] - 1) / 2, posc_px[1] + (res[1] - 1) / 2)
+        posccd = (posphy[0] / pxs[0], -posphy[1] / pxs[1])
+
+        if absolute:
+            # Convert into the referential with the top-left corner as origin
+            posccd = (posccd[0] + (res[0] - 1) / 2, posccd[1] + (res[1] - 1) / 2)
 
-        # Round to int, and clip to within CCD
-        posccd = (max(0, min(int(round(posccd[0])), res[0] - 1)),
-                  max(0, min(int(round(posccd[1])), res[1] - 1)))
+        if clip:
+            # Round to int, and clip to within CCD
+            posccd = (max(0, min(int(round(posccd[0])), res[0] - 1)),
+                      max(0, min(int(round(posccd[1])), res[1] - 1)))
 
-        if not 0 <= posccd[0] < res[0] or not 0 <= posccd[1] < res[1]:
-            logging.warning("Pos %s out of the CCD", posccd)
+            if not 0 <= posccd[0] < res[0] or not 0 <= posccd[1] < res[1]:
+                logging.warning("Pos %s out of the CCD", posccd)
```

Comment:
if you put this inside the `if  clip:` and after doing the actual clipping I think `not 0 <= posccd[0] < res[0] or not 0 <= posccd[1] < res[1]` will always be False. Or did I miss anything?

It might be nice to put this before the actual clipping and add that you will clip the data